### PR TITLE
Use ginko.junit-report in Makefile

### PR DESCRIPTION
**Pull request by danielerez**
Use --ginkgo.junit-report instead of the deprecated --ginkgo.reportFile.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,18 +14,18 @@ VERBOSE ?= false
...
 GO_UNITTEST_FLAGS = --format=$(GO_TEST_FORMAT) $(GOTEST_PUBLISH_FLAGS) -- -count=1 -cover -coverprofile=$(COVER_PROFILE)
-GINKGO_UNITTEST_FLAGS = -ginkgo.focus="$(FOCUS)" -ginkgo.v -ginkgo.skip="$(SKIP)" -ginkgo.v -ginkgo.reportFile=$(GINKGO_REPORTFILE)
+GINKGO_UNITTEST_FLAGS = -ginkgo.focus="$(FOCUS)" -ginkgo.v -ginkgo.skip="$(SKIP)" -ginkgo.v -ginkgo.junit-report=$(GINKGO_REPORTFILE)
 
 
 .PHONY: build
 
 build:
-	podman build -f Dockerfile.openshift-appliance . -t $(IMAGE)
+	docker build -f Dockerfile.openshift-appliance . -t $(IMAGE)
```
Comment (nmagnezi) [inline Makefile:28]: why this change?

Comment (danielerez) [reply Makefile:28]: by mistake:) fixing.
